src/TestUtils/src/UITest.Appium/Actions/AppiumScrollActions.cs

```diff
@@ -147,12 +149,13 @@ static void ScrollToLeft(AppiumDriver driver, AppiumElement element, ScrollStrat
 			int endX = (int)(position.X + (size.Width * swipePercentage));
 			int endY = startY;
 
-			new TouchAction(driver)
-				.Press(startX, startY)
-				.Wait(strategy != ScrollStrategy.Programmatically ? swipeSpeed : 0)
-				.MoveTo(endX, endY)
-				.Release()
-				.Perform();
+			OpenQA.Selenium.Appium.Interactions.PointerInputDevice touchDevice = new OpenQA.Selenium.Appium.Interactions.PointerInputDevice(PointerKind.Touch);
+			var scrollSequence = new ActionSequence(touchDevice, 0);
+			scrollSequence.AddAction(touchDevice.CreatePointerMove(CoordinateOrigin.Viewport, startX, startY, TimeSpan.Zero));
+			scrollSequence.AddAction(touchDevice.CreatePointerDown(PointerButton.TouchContact));
+			scrollSequence.AddAction(touchDevice.CreatePointerMove(CoordinateOrigin.Viewport, endX, endY, TimeSpan.FromMicroseconds(strategy != ScrollStrategy.Programmatically ? swipeSpeed : 0)));
```

Comment:
I think these timespans are too fast

If I slow down the move to a second then it scrolls just fine.

Reply:
Fixed.